src/common/autoconf/generated-configure.sh

```diff
@@ -41438,6 +41471,20 @@ $as_echo "$ac_cv_c_bigendian" >&6; }
         C_O_FLAG_NORM="-O2"
         C_O_FLAG_NONE="-O0"
       fi
+    elif test "x$TOOLCHAIN_TYPE" = xclang; then
+      if test "x$OPENJDK_TARGET_OS" = xmacosx; then
+        # On MacOSX we optimize for size, something
+        # we should do for all platforms?
+        C_O_FLAG_HIGHEST="-Os"
```

Comment:
This is weird. why does it emphasize on code size for MacOS?

Reply:
This is the same flag as using gcc on macOS. But I can run an experiment to see the diff in size by switching this flag to `-O3`.

Reply:
the tip is also same. 
https://hg.openjdk.java.net/jdk/jdk/file/7ef41e83066b/make/autoconf/flags-cflags.m4#l302